src/sparta/checkers/SimpleFlowAnnotatedTypeFactory.java

```diff
@@ -328,6 +328,22 @@ public AnnotationMirror getTopAnnotation(AnnotationMirror start) {
         }
 
         @Override
+        public AnnotationMirror getBottomAnnotation(AnnotationMirror start) {
+            if (start.toString().contains("Sink")) {
```

Comment:
Oh, from the code history, these two methods are created by two persons, so maybe they by nature used different way of deciding the type system.

To me it seems instanceof way should work on both methods.